src/playermat/Playermat.ttslua

```diff
@@ -916,6 +916,7 @@ end
 
 function onClick_hideOrShowOptions(player)
   GlobalApi.changeWindowVisibility(player.color, "optionPanelMain", _, self)
+  Wait.time(function() updateSave() end, 1)
```

Comment:
I think we want to narrow this down further and use frames as delay (afaik 3 is sufficient)

Reply:
fine with me